src/pynxtools/nomad/schema.py

```diff
@@ -105,8 +106,8 @@
     "NXsample": [CompositeSystem],
     "NXsample_component": [Component],
     "NXidentifier": [EntityReference],
-    "NXentry": [ActivityStep],
-    "NXprocess": [ActivityStep],
+    "NXentry": [ActivityStep],  # , Task],
```

Comment:
This should eventually becomes its own workflow (nested in the workflow of the whole nxs file) containing its own Tasks. But fine for now